File: 3040.py
#10c2 = 45 => 난쟁이 두명을 브루트포스로 배제할꺼임
input=open(0).readline
buf=[]
for _ in range(9): buf.append(int(input()))
for i in range(9):
    for j in range(i+1,9):
        res=[]
        sigma=0
        for k in range(9):
            if k==i or k==j: continue
            else: 
                sigma+=buf[k]
                res.append(buf[k])
                if sigma==100 and len(res)==7:
                    for element in res: print(element)
                    exit(0)

File: 3040.c
#include <stdio.h>
int main(){
    int buf[9];
    int res[7];
    int sigma;
    int entry=0;
    for (int i=0; i<9; i++) scanf("%d",&buf[i]);
    for (int i=0; i<9; i++){
        for (int j=i+1; j<9; j++){
            for (int idx=0; idx<7; idx++) res[idx]=0;
            sigma=0;
            entry=0;
            for (int k=0; k<9; k++){
                if (k==i || k==j) continue;
                else{
                    sigma+=buf[k];
                    res[entry]=buf[k];
                    if (sigma==100 && entry==6){
                        for (int l=0; l<7; l++) printf("%d\n",res[l]);
                        return 0;
                    }
                    entry+=1;
                }
            }
        }
    }
    return 0;
}
